manifests/share.pp

```diff
@@ -3,9 +3,9 @@
 # This class setups up a config file to be included into smb.conf
 #
 define winbind::share (
-  $ensure_setting  = present,
-  $settings_hash   = undef,
```

Comment:
Well I guess they don't necessarily _need to be in hiera_, but I guess I don't see any reason not to let users set default values in their hiera, rather than forcing them to change it in every `winbind::share` type they may have. Does that make sense or am I off my rocker?